scipy/special/add_newdocs.py

```diff
@@ -161,6 +161,62 @@ def add_newdoc(place, name, doc):
 
     """)
 
+
+add_newdoc("scipy.special", "agm",
+    """
+    agm(a, b)
+
+    Compute the arithmetic-geometric mean of `a` and `b`.
+
+    Start with a_0 = a and b_0 = b and iteratively compute::
+
+        a_{n+1} = (a_n + b_n)/2
```

Comment:
There is a long-standing, if unwritten, guideline that LaTeX notation should be restricted to the Notes section of the docstring.  This is implied in subsection 10 ("Notes") of the "Sections" section of the docstring standard, where it says "This section may include mathematical equations, written in LaTeX format".  One could interpret that to mean that LaTeX should not be used elsewhere.

As someone who frequently uses the non-rendered text form of the docstrings in ipython sessions, I agree with this guideline.  I really don't want LaTeX in the short summary description at the beginning of the docstring.  I love the rendered form generated by LaTeX, but I really dislike having to parse it myself, and we should not impose the burden of mentally parsing LaTeX on users.  Restricting `:math:` to the Notes seems like a reasonable compromise.

Now, `a_{n+1}` happens to be valid LaTeX, but I think it is also comprehensible to someone who has never used LaTeX.